streams/runners/storm/runtime/src/main/java/com/hortonworks/streamline/streams/runtime/storm/event/correlation/EventCorrelatingOutputCollector.java

```diff
@@ -14,9 +14,10 @@
  * limitations under the License.
  **/
 
-package com.hortonworks.streamline.streams.runtime.storm.testing;
+package com.hortonworks.streamline.streams.runtime.storm.event.correlation;
 
 import com.hortonworks.streamline.streams.StreamlineEvent;
+import com.hortonworks.streamline.streams.runtime.storm.testing.StormEventCorrelationInjector;
```

Comment:
nit: should this be moved to storm.event.correlation package ?

Reply:
Nice finding. Addressed.